drivers/hmis/app/controllers/hmis/sessions_controller.rb

```diff
@@ -22,6 +22,16 @@ def new
 
   # POST /hmis/login
   def create
+    # If the account has been locked, show an appropriate message. Ideally this would happen after the password is
+    # validated however at that point there doesn't seem to be a way to differentiate failures for bad credentials vs
+    # those due to locked accounts without performing a second validation (which could trigger unintended lockouts)
+    #
+    # There is a potential security issue in that this allows account emails to be enumerated
+    if sign_in_params['email'] && sign_in_params['password']
```

Comment:
Is this only relevant to Okta?  I feel like the existing warehouse returns the locked message correctly, but maybe that's not true on the HMIS side.  
I guess I don't have a big problem with this, you can already enumerate email addresses by guessing with incorrect passwords.  
We should confirm the `hmis/sessions*` paths are all in rack_attack similarly to how the main login forms are.

Reply:
This change is only relevant for email/password logins. Okta authentication doesn't use the sessions controller. Looks like the `hmis_sign_in_path` endpoint is protected by rack attack, just as we do for the warehouse side.